test/kubemark/resources/hollow-node_template.json

```diff
@@ -145,27 +157,48 @@
 				},
 				{
 					"name": "hollow-node-problem-detector",
-					"image": "gcr.io/google_containers/node-problem-detector:v0.2",
+					"image": "gcr.io/google_containers/node-problem-detector:v0.3",
```

Comment:
So IIUC, since v0.3 version of npd, we no longer are starting npd as part of the image, right? @Random-Liu